compress_raw_reports_telemetry.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# Deal with spaces in filenames:
+# https://www.reeltoreel.nl/wiki/index.php/Dealing_with_spaces_in_filenames
+# The default value of IFS is " \t\n" (e.g. <space><tab><newline>)
+SAVEIFS=$IFS
+IFS=$(echo -en "\n\b")
+
+path="/opt/telemetry/raw/"
+
+for f in `ls -1 $path | grep -v ".gz"`; do
+  echo $f
+  gzip $path/$f
+done
+
+IFS=$SAVEIFS
```

Comment:
the save and restore are only necessary if there's more shell code following, of course, but it can't hurt in case something gets added to this script I guess

Reply:
yep, this is why it's there